### DISCUSSION TITLE
GLIBC_2.32' not found

### DISCUSSION THREAD

**Discussion by N0o01** [projectdiscovery/naabu#794] [Q&A]
root@a~# unzip naabu_2.1.7_linux_amd64.zip 
Archive:  naabu_2.1.7_linux_amd64.zip
  inflating: naabu                   
root@ecs-ZAIW1:~# ls
naabu  naabu_2.1.7_linux_amd64.zip  resolv.conf
root@a~# chmod +x naabu
root@a~#./naabu -h
./naabu: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./naabu)
./naabu: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./naabu)

**Comment by MetzinAround** [comment]
Hi! 

If you run `sudo apt install libpcap-dev` that should install the needed dependency. You may need to reinstall naabu after. 

Let me know how the process goes!

**Reply by N0o01** [reply]
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libpcap-dev is already the newest version (1.8.1-6ubuntu1.18.04.2).
The following packages were automatically installed and are no longer required:
  libasprintf0v5 libbind9-140 libdbus-glib-1-2 libdns162 libgdbm3 libicu55 libisc160 libisccc140 libisccfg140 liblwres141 linux-headers-4.4.0-87 linux-headers-4.4.0-87-generic
  linux-image-4.4.0-87-generic linux-image-extra-4.4.0-87-generic
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

**Reply by MetzinAround** [reply]
Ok. 

Did you get the same error? Did you reinstall naabu?

**Comment by ehsandeep** [comment]
See https://github.com/projectdiscovery/naabu/discussions/696
